sensu/goplugin_test.go

```diff
@@ -61,6 +62,14 @@ func TestSetOptionValue_FalseBool(t *testing.T) {
 	assert.Equal(t, false, finalValue)
 }
 
+func TestSetOptionValue_Int64(t *testing.T) {
+	var value int64
+	option := defaultOption1
+	option.Value = &value
+	assert.NoError(t, setOptionValue(&option, "42"))
+	assert.Equal(t, value, int64(42))
+}
+
```

Comment:
Ultimately, maybe we should add test of every single one of these types, to make sure we don't inadvertently break any of them during a refactor?